webapp/src/main/resources/public/js/app.js

```diff
@@ -146,30 +146,34 @@ function startApp(messages) {
                 // re-authenticate and the load the API response instead of showing the frontend
                 window.location.href = UrlHelper.getUrlWithContextPath(currentLocation);
```

Comment:
when i tried that path -dev, it seemed to go back to the repository page? given the comment I was wondering if it will go to the API instead. Have you done more testing around that?